### ISSUE TITLE
Deep Learning Weekly - ? - Issue #83: Nature ML PyTorch 1.0 TPU vs V100 L2 vs Weight Decay MLPerf Benchmark and Swift for TensorFlow...

### ISSUE THREAD

**Issue by ikedaosushi** [ikedaosushi/tech-news#3173]
Deep Learning Weekly - ? - Issue #83: Nature ML, PyTorch 1.0, TPU vs V100, L2 vs Weight Decay, MLPerf Benchmark and Swift for TensorFlow...<br>
<br>
https://ift.tt/2G148cc
